pkg/cluster/spec.go

```diff
@@ -491,6 +487,28 @@ func userAgent(eksAComponent, version string) string {
 	return fmt.Sprintf("eks-a-%s/%s", eksAComponent, version)
 }
 
+type EksdRelease struct {
+	ReleaseManifestContent []byte
+	ReleaseCrdContent      []byte
+}
+
+func (s *Spec) GetEksdRelease(release v1alpha1.EksDRelease) (*EksdRelease, error) {
```

Comment:
This method signature is a bit confusing

Should this be something like

```suggestion
func (s *Spec) ReadEksdManifests(release v1alpha1.EksDRelease) (*EksdManifests, error) {
```